src/vortex.py

```diff
@@ -15,3 +15,20 @@ def get_env(self):
         '''
         pass
 
+    def register_endpoint(self, func):
+        '''
+        Registers an endpoint method to track the data flowing through its response.
+        Usage example - 
+        ```python
+        @app.GET
+        @vortex.register_endpoint
+        def response_function(...):
+            ...
+        ```
+        '''
+        def register(*args, **kwargs):
+            out = func(*args, **kwargs)
+            self.endpoints.add(out)
```

Comment:
I think since the functions added to `self.endpoints` will never be executed via `self.endpoints`, you can simply add just the name of the function rather than the entire function.

Reply:
This is going to add whatever the response of the route is to `self.endpoints`
That is necessary because `self.endpoints` is our sink and responses need a data path to it

Reply:
So we are going to check the response body for environment  variables?

Reply:
Yes. A more accurate description is we're going to check the data that is being returned by an endpoint function before it gets converted into a proper HTTP response.

Reply:
LGTM. After further development, we might come back and tweak this one a little if necessary. Marking as resolved.